CMakeLists.txt

```diff
@@ -94,3 +98,37 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
+if(UNIX)
+    # General settings
+    set(CPACK_GENERATOR "TBZ2")
+    set(CPACK_PACKAGE_NAME "libqhotkey")
+    set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Global hotkey library for Qt software")
+    set(LICENSE_FILE "LICENSE")
```

Comment:
This variable is never used.

Reply:
That's a leftover from my attempt to build an RPM, but I couldn't get it working.
Line removed.